src/pages/_app.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react'
+import App, { Container } from 'next/app'
+import { ThemeProvider } from 'styled-components'
+import theme from '../styles/theme'
+import '../styles/reset.css'
+import '../styles/style.css'
+
+type Props = {
+  children?: React.Node
+}
+
+class Layout extends React.Component<Props> {
+  render() {
+    const { children } = this.props
+
+    return <div className="layout">{children}</div>
```

Comment:
Just curious, do we acutally need that class?

Reply:
Nope, we don't 😅